### ISSUE TITLE
Har file fails to transform to py file

### ISSUE THREAD

**Issue by mjohnson767** [zalando-incubator/transformer#68]
**Describe the bug**
I have a few .har files that won't covert to .py files using transformer. Reason unknown. I can see the attempt to convert it but after-the-fact the .py file size is 0 bytes. 

**Transformer version**
Output of the command `transformer --version`.
1.2.6

**To Reproduce**
Steps and input files to reproduce the behavior:
1. Create the file "..."
transformer C:\Users\mjohns33\PycharmProjects\Master\ppmi_v2.har >newppmo.py
2. Run Transformer with the arguments "..."
No arguments.
3. See error
No errors, it appears to have ran but nothing is generated on the .py file.

**Expected behavior**
The .har file to be converted to .py file

**Screenshots**

**Desktop (please complete the following information):**
-Win10 

**Additional context**
I'd like to try to play back the .har file using locust but I'm unsure of the syntax in the locustfile.py. 

[ppmi_v2.zip](https://github.com/zalando-incubator/transformer/files/4372250/ppmi_v2.zip)
 other context about the problem here.

**Comment by thilp** [comment]
Hi @mjohnson767, thank you for reporting the problem!

This might be related to running on Windows, as I can't seem to be able to reproduce the problem on Linux:

```
$ curl -LO https://github.com/zalando-incubator/transformer/files/4372250/ppmi_v2.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   576  100   576    0     0   3490      0 --:--:-- --:--:-- --:--:--  3490
100 2155k  100 2155k    0     0   840k      0  0:00:02  0:00:02 --:--:-- 1113k

$ unzip ppmi_v2.zip
Archive:  ppmi_v2.zip
  inflating: ppmi_v2.har

$ pipx run --spec har-transformer transformer --version
1.2.6

$ pipx run --spec har-transformer transformer ppmi_v2.har >test.py
2020-03-24 14:08:27,804 INFO    No ppmi_v2.weight provided for ppmi_v2.har: assigning default weight 1 ([Errno 2] No such file or directory: 'ppmi_v2.weight')

$ head test.py
# File automatically generated by Transformer v1.2.6:
# https://github.com/zalando-incubator/Transformer
import re
from locust import HttpLocust
from locust import TaskSequence
from locust import TaskSet
from locust import seq_task
from locust import task
class _tmp_ppmi_v2_har_829294038(TaskSequence):
    @seq_task(1)
```

Here is the full `test.py` that was produced: [test.py.zip](https://github.com/zalando-incubator/transformer/files/4375418/test.py.zip).

I'll try again on Windows as soon as I can, but I don't have a development environment there, so that might take some time. If you have any more clues, I'll gladly hear about them!

Have a nice day!

**Comment by mchapko** [comment]
hi, Same here, har file was stored on windows with google chrome.
With running "transformer file.har  > file.py" i see an error with decoding. 
Traceback (most recent call last):
  File "[MASKED]\lib\site-packages\transformer\scenario.py", line 316, in from_har_file
    har = json.load(file)
  File "[MASKED]\python39\lib\json\__init__.py", line 293, in load
    return loads(fp.read(),
  File "[MASKED]\python39\lib\encodings\cp1250.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 251566: character maps to <undefined>

**Comment by mchapko** [comment]
as workaround i have modified locally:
In:
 @classmethod
    def from_har_file
...    
 try:
            with path.open(**encoding="utf8"**) as file:


And file was generated properly

**Comment by mjohnson767** [comment]
I've installed har-transformer via pip; is the change needed in the actual har-transformer.har file?
Can you provide the steps to fix?
Great find.

**Comment by mchapko** [comment]
Hi @mjohnson767, actually you need to do modification in your package installed in python directory.
[your python directory] \lib\site-packages\transformer\scenario.py
Find mentioned file and modify the file line as mentioned above. Nothing more is needed.
Then try to run transformer.
This is workaround only to point where is the issue.
Fix is needed from transformer team as requires some more coding to be bulletproof :)

**Comment by thilp** [comment]
I believe this should be addressed by #79, i.e. v2.0.0. Please let me know if that is not the case. :slightly_smiling_face:

**Comment by parohanoid** [comment]
Hi @thilp 

Getting this error for fiddler generated .har files:
`WARNING while searching for HAR files, skipping {file_name}: Unexpected UTF-8 BOM (decode using utf-8-sig): line 1 column 1 (char 0)`

Thanks to the comments I was able to fix it locally by changing 
`encoding="utf-8"` to `encoding="utf-8-sig"`

inside

```
def from_har_file
...
try:
with path.open(encoding="utf-8-sig") as file:
```


This is just an FYI in case others want to fix it

Awesome tool btw!
